README.md

```diff
@@ -129,6 +129,80 @@ file { '/etc/tmpfiles.d/foo.conf':
 ~> Class['systemd::tmpfiles']
 ```
 
+### timer units
+Create a timer unit and a service unit to execute from
+that timer at the same time. The following will manage timer unit 
+service and enable it.
+
+```puppet
+systemd::timer{'runoften.timer':
+  timer_source   => "puppet:///modules/${module_name}/runoften.timer",
+  service_source => "puppet:///modules/${module_name}/runoften.service",
+  active         => true,
+  enable         => true,
+}
+```
+or a trivial daily run and manage the timer unit manually
+
+```puppet
+$_timer = @(EOT)
+[Timer]
+OnCalendar=daily
+RandomizedDelaySec=1d
+EOT
+
+$_service = @(EOT)
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/touch /tmp/file
+EOT
+
+if versioncmp($facts['puppetversion'],'6.1.0') < 0 {
```

Comment:
When using `enable` and/or `active` I think that it already takes care of this, right? Might be good to explain this to the user.

Reply:
Yes, I think so.  This adds a `service` resource and the required (for puppet 5) relationship between daemon reload and the service.

Reply:
Link added:

```
@see https://www.freedesktop.org/software/systemd/man/systemd.timer.html systemd.timer(5)
```
seems to be the correct syntax.